README.md

```diff
@@ -57,7 +57,7 @@ Work includes:
 
 ## Archives
 
-As part of creating the ECMAScript Modules implementation for Node.js 12.0.0, [use cases](./doc/use-cases.md) were brainstormed, which led to a list of [features](./doc/features.md), which guided the [plan for the implementation](./doc/plan-for-new-modules-implementation.md).
+- **[`features.md`](./doc/archive/features.md)**: As part of creating the ECMAScript Modules implementation for Node.js 12.0.0, [use cases](./doc/use-cases.md) were brainstormed, which led to a list of features. This feature list, which previously existed as a section in this repo's root README, informed what became the [plan for the implementation](./doc/plan-for-new-modules-implementation.md).
```

Comment:
If you're going to make a bullet for each archived file, then we need a bullet for the `plan-for-new-modules-implementation.md` file too. It's also archived, and both files should be in `doc/archive`.